cypress/e2e/supervision/clients/edit-client.cy.js

```diff
@@ -9,17 +9,20 @@ const editClient = (isCourtReferenceChanged) => {
       method: "PUT",
       url: `/supervision-api/v1/clients/${id}`,
     }).as("editClientCall");
+    cy.wait(2000);
```

Comment:
Is there any way to do this without a `cy.wait()`? I think `cy.get()` should retry until the element it is looking for can be selected anyway.

Reply:
Thanks.